.github/workflows/add-labels-priority.yml

```diff
@@ -8,18 +8,27 @@ jobs:
     runs-on: ubuntu-latest
     if: github.event.action == 'opened'
     steps:
+      - uses: actions/create-github-app-token@v1
+        id: app-token
+        with:
+          app-id: ${{ vars.BOT_APP_ID }}
+          private-key: ${{ secrets.BOT_PRIVATE_KEY }}
       - uses: simple-icons/labeler@v1
         with:
-          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          repo-token: ${{ steps.app-token.outputs.token }}
 
   assign-to-project:
     runs-on: ubuntu-latest
     name: Assign to Project
     if: github.event.action == 'opened'
     needs: triage
-    env:
-      MY_GITHUB_TOKEN: ${{ secrets.AUTO_ASSIGN_WORKFLOW_TOKEN }}
```

Comment:
I think that the next permission must be configured on the app too at organization level rather than at repository level.

![image](https://github.com/simple-icons/simple-icons/assets/23049315/0b3e23f7-ee0f-40ad-bc66-dc82535a3201)

Reply:
Agree.